model-optimizer/extensions/ops/ReduceOps.py

```diff
@@ -138,11 +140,16 @@ class ReduceMean(ReduceOp):
     enabled = True
 
 
-class ReduceL2(ReduceOp):
-    op = 'ReduceL2'
-    op_type = None
+class ReduceLp(ReduceOp):
```

Comment:
Please, create a new operation AttributedReduceLp (where `p` is an attribute) and implement transformation which converts it to ReduceLp (with `p` as input).